zipline/data/ffc/datasets/README.md

```diff
@@ -0,0 +1,5 @@
+Currently USEquityPricing is located in zipline.data.pricing
```

Comment:
I have this in zipline.data.equities on my current branch, but there's also a zipline.data.ffc submodule that could be more appropriate.